addresses/420.json

```diff
@@ -2,7 +2,7 @@
   "CONTRACT_1155_IMPL": "0x363319e62B5D68AA5152C6C885f68Bec2f065f83",
   "FACTORY_IMPL": "0xdb6576BD7EAFB2D38826D614Ec0B908cCCCa3F48",
   "FACTORY_PROXY": "0xb0C56317E9cEBc6E0f7A59458a83D0A9ccC3e955",
-  "FIXED_PRICE_SALE_STRATEGY": "0x9168C5ba5a0a76db8A1BF5b2eE5557f2A0ECA4f4",
+  "FIXED_PRICE_SALE_STRATEGY": "0x5b31082c269e1e575b371e14dec2594ebc6a4c5a",
```

Comment:
@iainnash this test actually exposed that this value was configured incorrectly; same with below (for chainConfigs)

Reply:
amazing, thanks!